### ISSUE TITLE
$scroll

### ISSUE THREAD

**Issue by otomiewogift** [scroll-tech/contribute-to-scroll#1675]
### Name of the project

Scroll

### Project type

Library

### Site of the project

Scroll.io

### Github repository of the project

https://scroll.io/blog/contribute-to-scroll

### Networks

Scroll Mainnet

### Additional context

Integration of V3 to scroll mainnet bridge for cheaper fees

**Comment by BLIZSAY** [comment]
Yes I am having trouble issue with also, thanks for adressing it!
